skaffold.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: skaffold/v4beta6
+kind: Config
+metadata:
+  name: turbinia
+build:
+  local:
+    useDockerCLI: true
+    useBuildkit: true 
+    concurrency: 0
+  artifacts:
+    # Uncomment below when developing on the Turbinia Worker
+    # - image: turbinia-worker
+    #   docker:
+    #     dockerfile: docker/worker/Dockerfile
+    #     buildArgs:
+    #       TURBINIA_DEBUG: 1
+    #       TURBINIA_HOTRELOAD: 1
+    #       TURBINIA_DEBUG_PORT: 10000
+    #   sync:
+    #     infer:
+    #     - turbinia/**
+
+    # Uncomment below when developing on the Turbinia Server
+    - image: turbinia-server
+      docker:
+        dockerfile: docker/server/Dockerfile
+        buildArgs:
+          TURBINIA_DEBUG: 1
+          TURBINIA_HOTRELOAD: 1
+          TURBINIA_DEBUG_PORT: 20000
+      sync:
+        infer:
+        - turbinia/**
+deploy:
```

Comment:
One annoyance was that each rebuild also rebuilds the Redis chart and that takes longer then the other pods to be ready so deployment initially fails. Not sure if there is a way to prevent redeployment of sub dependencies?

Reply:
Also disregard deployment much faster when checking your branch out, I think maybe had to do with .dockerignore